src/Core/Ast/Parser/NikicPhpParser/FileReferenceVisitor.php

```diff
@@ -131,10 +143,11 @@ private function enterClassLike(ClassLike $node): void
         if (null !== $name) {
             $isInternal = false;
             if (null !== $docNodeCrate) {
-                $isInternal = [] !== array_merge(
-                    $docNodeCrate[0]->getTagsByName('@internal'),
-                    $docNodeCrate[0]->getTagsByName('@deptrac-internal')
-                );
+                $isInternal = [] !== $docNodeCrate[0]->getTagsByName('@deptrac-internal');
```

Comment:
The intention of this code is a bit less clear after the rewrite. Can we make it more clear/declarative?

Reply:
I changed it again in the follow-up, that should make it clear again.

Reply:
From the dsicussion with @gennadigennadigennadi welow, it sounds like we don't want to uuse both tags at once. What do you think?

Reply:
I think we should keep the option to support `@deptrac-internal` and any user-specified tag. `@deptrac-internal` is always valid, it never has any other meaning as it is tool prefixed tag. If the user wants to specify another tag (like `@internal` or anything else) that's up to them.